test/src/test/java/org/corfudb/runtime/object/CompileProxyTest.java

```diff
@@ -376,7 +376,7 @@ public void testObjectCompoundWriteConcurrency() throws Exception {
         scheduleConcurrently(concurrency, t -> {
             sharedCorfuCompound.set(sharedCorfuCompound.new Inner("A"+t, "B"+t), t);
         });
-        executeScheduled(concurrency, PARAMETERS.TIMEOUT_SHORT);
+        executeScheduled(concurrency, PARAMETERS.TIMEOUT_NORMAL);
```

Comment:
Changing timeout from 1 sec to 10 sec would add into serial execution execution of test?

Reply:
No, it just doesn't cancel the ongoing task execution for up to 10 seconds, it doesn't affect the concurrency or interfere with the task execution.
The flakiness is due to some of the tasks taking more than 1 second in the `executeScheduled` call.

This can also be verified if we put a `Thread.sleep(2000)` (2 seconds) inside the task, which consistently results in the same failure -- `Cancellation Exception` as the task doesn't complete within 1 second.